db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20141010131030) do
+ActiveRecord::Schema.define(version: 20141015152508) do
```

Comment:
The schema file defines the current state of the data base (PostgreSQL or SQLite). It auto-updates every time a new migration is written and run on the data base. It gets reverted when we roll back the data base.